httpclient-backend/zio/src/main/scala/sttp/client/httpclient/zio/HttpClientZioBackend.scala

```diff
@@ -0,0 +1,62 @@
+package sttp.client.httpclient.zio
+
+import java.net.http.{HttpClient, HttpRequest}
+
+import sttp.client.httpclient.HttpClientBackend.EncodingHandler
+import sttp.client.httpclient.{HttpClientAsyncBackend, HttpClientBackend}
+import sttp.client.impl.zio.RIOMonadAsyncError
+import sttp.client.{FollowRedirectsBackend, SttpBackend, SttpBackendOptions}
+import zio._
+
+class HttpClientZioBackend private (
+    client: HttpClient,
+    closeClient: Boolean,
+    customizeRequest: HttpRequest => HttpRequest,
+    customEncodingHandler: EncodingHandler
+) extends HttpClientAsyncBackend[Task, Nothing](
+      client,
+      new RIOMonadAsyncError,
+      closeClient,
+      customizeRequest,
+      customEncodingHandler
+    )
+
+object HttpClientZioBackend {
+  private def apply(
+      client: HttpClient,
+      closeClient: Boolean,
+      customizeRequest: HttpRequest => HttpRequest,
+      customEncodingHandler: EncodingHandler
+  ): SttpBackend[Task, Nothing, Nothing] =
```

Comment:
Why don't we have WebSockets, afaik they don't require zio-streams. Do they?

Reply:
I wanted to make the PR as small as possible, and add web sockets in the next one